src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -260,6 +260,11 @@ QString toUnicode(const std::string& toConvert) {
             ->toUnicode(toConvert.data(), static_cast<int>(toConvert.length()));
 }
 
+QString convertComment(const std::string& toConvert) {
+    int length = static_cast<int>(toConvert.length()) - 2; // strip off trailing nullbyte
+    return QTextCodec::codecForName("UTF-16BE")->toUnicode(toConvert.data(), length);
+}
```

Comment:
std::string the wrong type here. It is a non null terminated char string. Very misleading.
It is used here from the Kaitai code generator we cannot change. But we should make it clear.

Proposal: 
```suggestion
QString formKaitaiUtf8BeString (const std::string& toConvert) {
    // Kaitai uses std::string as single container for all string encodings. 
    int length = static_cast<int>(toConvert.length()) - 2; // strip off trailing nullbyte
    return QTextCodec::codecForName("UTF-16BE")->toUnicode(toConvert.data(), length);
}
```